lang/performous.pot

```diff
@@ -17,647 +17,361 @@ msgstr ""
-msgid "Hide pitch wave, notes and scoring."
-msgstr ""
-
-#: ../data/config/schema.xml:56
-msgid "Lyric highlight style"
-msgstr ""
-
-#: ../data/config/schema.xml:60
-msgid "Autoplay"
-msgstr ""
-
-#: ../data/config/schema.xml:61
-msgid "Keep playing random songs after the playlist is empty."
-msgstr ""
-
-#: ../data/config/schema.xml:64
-msgid "Pitch waves"
-msgstr ""
-
-#: ../data/config/schema.xml:65
-msgid "Enable singing pitch display (when not in karaoke mode)."
-msgstr ""
-
-#: ../data/config/schema.xml:70
-msgid "Theme"
-msgstr ""
-
-#: ../data/config/schema.xml:71
-msgid "Name of the theme to use."
-msgstr ""
-
-#: ../data/config/schema.xml:75
-msgid "Playlist screen timeout"
-msgstr ""
-
-#: ../data/config/schema.xml:76
-msgid ""
-"How long it will take before the next song in the playlist starts "
-"automatically."
-msgstr ""
-
-#: ../data/config/schema.xml:84
-msgid "Webserver access"
-msgstr ""
-
-#: ../data/config/schema.xml:85
-msgid ""
-"Define if you want to control performous remotely with web interface "
-"(requires restart)."
-msgstr ""
-
-#: ../data/config/schema.xml:90
-msgid "Webserver port"
-msgstr ""
-
-#: ../data/config/schema.xml:91
-msgid ""
-"Port to use for the web server. Choose one that is not used by other "
-"applications."
-msgstr ""
-
-#: ../data/config/schema.xml:94
-msgid "Keyboard as guitar"
-msgstr ""
-
-#: ../data/config/schema.xml:95
-msgid "Enable keyboard as guitar (Frets on Fire mode)."
-msgstr ""
-
-#: ../data/config/schema.xml:98
-msgid "Keyboard as drumkit"
-msgstr ""
-
-#: ../data/config/schema.xml:99
-msgid "Enable keyboard as drumkit."
-msgstr ""
-
-#: ../data/config/schema.xml:102
-msgid "Keyboard as dance pad"
-msgstr ""
-
-#: ../data/config/schema.xml:103
-msgid "Enable keyboard as dance pad."
-msgstr ""
-
-#: ../data/config/schema.xml:106
-msgid "Keyboard as keytar"
-msgstr ""
-
-#: ../data/config/schema.xml:107
-msgid "Enable keyboard as keytar."
-msgstr ""
-
-#: ../data/config/schema.xml:114
-msgid "Force controller A's type"
-msgstr ""
-
-#: ../data/config/schema.xml:115
-msgid ""
-"Override autodetection and force the first controller to the given type."
-msgstr ""
-
-#: ../data/config/schema.xml:122
-msgid "Force controller B's type"
-msgstr ""
-
-#: ../data/config/schema.xml:123
-msgid ""
-"Override autodetection and force the second controller to the given type."
-msgstr ""
-
-#: ../data/config/schema.xml:130
-msgid "Force controller C's type"
-msgstr ""
-
-#: ../data/config/schema.xml:131
-msgid ""
-"Override autodetection and force the third controller to the given type."
-msgstr ""
-
-#: ../data/config/schema.xml:138
-msgid "Force controller D's type"
-msgstr ""
-
-#: ../data/config/schema.xml:139
-msgid ""
-"Override autodetection and force the fourth controller to the given type."
-msgstr ""
-
-#: ../data/config/schema.xml:146
-msgid "Joystick configuration"
-msgstr ""
-
-#: ../data/config/schema.xml:147
-msgid "Force joystick to some kind of instrument."
```

Comment:
Are these deletions valid?

Reply:
hmmm.. I used poedits feature to update pot from source files. But it seems xml is not recognized. How do you guys refresh `performous.pot` normally?
```
xgettext -o performous.pot -j ../game/*.cc ../data/config/schema.xml
xgettext: warning: file '../data/config/schema.xml' extension 'xml' is unknown; will try C
```

Reply:
Found everything I needed in `README`, please see the updated commit